python/bottle/webappWithTests/Tests/functional_tests/test_functional.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from selenium import webdriver
+import unittest
+import os
+import sys
+import pytest
+
+class FunctionalTests(unittest.TestCase):
+
+	def setUp(self):
+		options = webdriver.ChromeOptions()
+		options.add_argument('--no-sandbox')
+		self.driver = webdriver.Chrome(os.environ["ChromeWebDriver"], chrome_options=options)
+
+	def test_selenium(self):
+		try:
+			webAppUrl = pytest.config.getoption('webAppUrl')
+			response = self.driver.get(webAppUrl)
+			html_source = self.driver.page_source
+			self.assertIn("<title>Home Page - Python Bottle Application</title>", html_source)
+		except AssertionError:
+			raise
+		except Exception as e:
+			sys.stderr.write('tests_selenium.Error occurred while executing tests: ' + str(e))
```

Comment:
when this happens the test fails right?

Reply:
asking since i don't see a _raise_ here. Just double check if writing to stderr fails the test